scripts/install_third_party_libs.py

```diff
@@ -220,71 +219,63 @@ def ensure_system_python_libraries_are_installed(package, version):
 
 
 def rewrite_android_proto_files():
-    """Edit all android proto files."""
-    # Since there is no simple configuration for imports when using protobuf to
-    # generate Python files we need to manually fix the imports.
-    # See: https://github.com/protocolbuffers/protobuf/issues/1491
-    protobuf_dir = (
-        pathlib.Path(
-            os.path.join(
-                common.THIRD_PARTY_DIR,
-                'oppia-proto-api-8f3cde883c31785438e80656a5b6bb26bd01b6a1'))
-            .glob('**/*.proto'))
-    for p in protobuf_dir:
-        if p.suffix == '.proto':
+    """Android proto files are the part of oppia-proto-api-* directory in the
+    third_party_libs directory. Currently, proto files generation doesn't
+    support importing other proto files from a different directory.
+    Because of it, we had to keep all the proto files in the same directory.
+    Since there is no simple configuration for imports when using protobuf to
+    generate Python files we need to manually fix the imports.
+    See: https://github.com/protocolbuffers/protobuf/issues/1491
+    """
+    protobuf_dir = pathlib.Path(
+        common.THIRD_PARTY_OPPIA_ANDROID_PROTO_DIR).glob('**/*.proto')
+    for file in protobuf_dir:
+        if file.suffix == '.proto':
             # Remove package statement.
             # Example: 'package org.oppia.proto.v1.api'.
             common.inplace_replace_file(
-                p.absolute(),
+                file.absolute(),
                 r'^package ([^\s]+)',
                 r'')
             # Remove option statement.
             # Example: 'option java_package = "org.oppia.proto.v1.api";'.
             common.inplace_replace_file(
-                p.absolute(),
+                file.absolute(),
                 r'^option java_.+',
                 r'')
             # Update import statement.
             # Example: 'import "org/oppia/proto/v1/api/state.proto";' to
             # 'import "state.proto";'.
             common.inplace_replace_file(
-                p.absolute(),
+                file.absolute(),
                 r'^import (?!\"google)([^\s]+)[\\/]([^\s]+)',
                 r'import "\2')
             # Remove all subpackage directories.
             # Example: 'org.oppia.'.
             common.inplace_replace_file(
-                p.absolute(),
+                file.absolute(),
                 r'org.oppia.*\.',
                 r'')
 
 
 def move_all_proto_files_to_third_party():
-    """Move all proto files from subdirectories
-    to the third_party folder.
+    """Move all the proto files from subdirectories of oppia-proto-api-*
+    to its parent folder. Currently, proto files generation doesn't
+    support importing other proto files from a different directory.
+    Because of it, we had to keep all the proto files in the same directory.
```

Comment:
Please do not copy comments wholesale between functions. Figure out which part of the comment should go in this function vs the above function, and put those parts in their proper places.

Reply:
Updated.